lib/purescript-ast/src/Language/PureScript/AST/Declarations.hs

```diff
@@ -223,29 +223,6 @@ data ExportSource =
   }
   deriving (Eq, Ord, Show, Generic, NFData, Serialise)
 
--- enable sorting lists of explicitly imported refs when suggesting imports in linting, IDE, etc.
--- not an Ord because this implementation is not consistent with its Eq instance.
--- think of it as a notion of contextual, not inherent, ordering.
```

Comment:
I don't think we need this comment anymore, since it now follows the order of the constructors.